src/ar_manage_peers.erl

```diff
@@ -43,7 +42,9 @@ format_stats(Peer, Perf) ->
 %% Peers who have behaved well in the past are favoured in ranking.
 %% New, unknown peers are given 100 blocks of grace.
 update(Peers) ->
-	ar_meta_db:remove_old(os:system_time(seconds)),
+	%% Remove performance records created more than a certain amount of time. It
```

Comment:
A suggestion: "Remove performance records **not updated for** more than a certain amount of time. Also, I do not understand what "how actively updated it is" means in this context.

Reply:
It was a misunderstanding this function does something unexpected. That's not the case, so I removed the comment. Looks ok?

Reply:
No text => no questions :)